firebase_admin/_utils.py

```diff
@@ -120,9 +139,9 @@ def handle_operation_error(error):
             message='Unknown error while making a remote service call: {0}'.format(error),
             cause=error)
 
-    status_code = error.get('code')
+    rpc_code = error.get('code')
```

Comment:
Doesn't this response contain a `status` field? That's how we map errors sent by other OnePlatform APIs. See `_handle_func_requests` function in this module.

https://cloud.google.com/apis/design/errors#http_mapping

Reply:
It does for non-operations responses. But for Operation errors it does not. Just message and code (details optional and not applicable in this case)